swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/securitygroups/AmazonSecurityGroupHandler.kt

```diff
@@ -109,6 +109,14 @@ class AmazonSecurityGroupHandler(
     }
   }
 
+  private fun determineApp(resource: Resource): String {
```

Comment:
Can this be moved to the abstract class to avoid duplication across handlers?

Reply:
Good idea - I'll do that.